### PR TITLE
App Submission - Squeak Road

### PR DESCRIPTION
**Pull request by yzernik**
# App Submission

### App name
Squeak Road

### 256x256 SVG icon
https://svgshare.com/s/jSc

...

### Gallery images
Screenshot at 2022-07-23 03-27-47
Screenshot at 2022-07-25 00-23-42
Screenshot at 2022-07-23 19-46-35
Screenshot at 2022-07-23 19-47-06
Screenshot at 2022-07-23 19-47-53

...

### I have tested my app on:
- [ ] [Umbrel dev environment](https://github.com/getumbrel/umbrel-dev)
- [x] [Umbrel OS on a Raspberry Pi 4](https://github.com/getumbrel/umbrel-os)
- [ ] [Custom Umbrel install on Linux](https://github.com/getumbrel/umbrel#-installation)

### CODE REVIEW

File: squeakroad/umbrel-app.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+repo: https://github.com/yzernik/squeakroad
+support: https://github.com/yzernik/squeakroad/discussions
+port: 8200
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+path: ""
+defaultUsername: ""
+defaultPassword: ""
```
Comment (lukechilds) [inline squeakroad/umbrel-app.yml:28]: We need to add this to the manifest so users can see the admin password.

```suggestion
defaultUsername: "umbrel"
deterministicPassword: true
```

File: squeakroad/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_SQUEAKROAD_IP
+      APP_PORT: $APP_SQUEAKROAD_PORT
```
Comment (lukechilds) [inline squeakroad/docker-compose.yml:7]: We want to bypass Umbrel auth for this app right? Sp people who don't have access to the Umbrel dashboard are still able to access the app? If so we need to add this:

```suggestion
      APP_PORT: $APP_SQUEAKROAD_PORT
      PROXY_AUTH_ADD: "false"
```

Comment (yzernik) [reply squeakroad/docker-compose.yml:7]: Yes, that's right.

File: squeakroad/umbrel-app.yml (hunk 1 above)
Comment (yzernik) [reply squeakroad/umbrel-app.yml]: Thanks.

**Comment by yzernik** [comment]
Thanks for the review @lukechilds 

I merged your suggestions, I also updated the app description to include a security warning.

**Review by lukechilds — APPROVED** [review]

**Comment by lukechilds** [comment]
Working great for me, awesome work @yzernik!

@mayankchhabra are you able to put together some gallery assets for this?

**Comment by lukechilds** [comment]
@yzernik we have gallery assets ready. You mentioned in DMs you have some things you wanna fix before release. Feel free to ping us here whenever you're ready to ship!

**Comment by mayankchhabra** [comment]
Great work once again @yzernik!! The app store gallery images and app icon are now live via [getumbrel/umbrel-apps-gallery](https://github.com/getumbrel/umbrel-apps-gallery/tree/master/squeakroad).

Good luck with the launch in the app store! 🚀

**Pull request merged**